lib/mixins/listen-on-root.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Issue #569: collapse::toggle::state triggered too many times
+ * @link https://github.com/bootstrap-vue/bootstrap-vue/issues/569
+ */
+export default {
+    data() {
```

Comment:
Also, where using Vue here, not Angular or React—if nothing is being rendered by that private prop, Vue won't do anything! Gotta love it